src/accelerate/test_utils/other.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+import os
+from pathlib import Path
+from typing import Callable, Dict
+
+import torch
+
+from ..logging import get_logger
+from ..state import PartialState
+
+
+# We must initialize the accelerate state before using the logging utility.
+state = PartialState()
+logger = get_logger(__name__)
```

Comment:
This is not allowed, any import to `test_utils` will init the PartialState. Get the logger, and then init the state in the test